spec/lib/science_wire_harvester_spec.rb

```diff
@@ -384,15 +384,15 @@ def setup(threshold)
       expect(science_wire_harvester).to receive(:add_contribution_for_harvest_suggestion).twice.and_call_original
       expect(science_wire_harvester).to receive(:create_contrib_for_pub_if_exists_by_author_ids).and_call_original
       expect(auth.publications.count).to eq(0)
-      science_wire_harvester.harvest_sw_pubs_by_wos_id_for_author('gorin', ['000224492700003'])
+      science_wire_harvester.send(:harvest_sw_pubs_by_wos_id_for_author, 'gorin', ['000224492700003'])
       auth.reload
       expect(auth.publications.count).to eq(1)
       expect(SciencewireSourceRecord.count).to eq(1)
       expect(PubmedSourceRecord.count).to eq(0)
       # Harvest the same publication and it should not duplicate.
       expect(science_wire_harvester).not_to receive(:create_contrib_for_pub_if_exists_by_author_ids)
       expect(science_wire_harvester).not_to receive(:create_or_update_pub_and_contribution_with_harvested_sw_doc)
-      science_wire_harvester.harvest_sw_pubs_by_wos_id_for_author('gorin', ['000224492700003'])
+      science_wire_harvester.send(:harvest_sw_pubs_by_wos_id_for_author, 'gorin', ['000224492700003'])
```

Comment:
Why is this an improvement?

Reply:
As the commit message describes, this is effectively an internal method: nothing outside the class calls it except for the test.  Obviously the test should not dictate the design of the class, so this allows the method to be marked `private` and still use the same tests.